profile_bluesky/startup/instrument/devices/magnets.py

```diff
@@ -0,0 +1,38 @@
+"""
+Magnet motors
+"""
+
+__all__ = [
+    'mag6t',
+    ]
+
+from ..session_logs import logger
+logger.info(__file__)
+
+from ophyd import Component, Device, EpicsMotor, FormattedComponent
+
+## Magnet and sample motors ##
+class Magnet6T(Device):
+
+    ## Motors ##
+    tabth = Component(EpicsMotor,'m53', labels=('motor','6T magnet'))  # 4T Mag Th
```

Comment:
each label should not have embedded whitespace (or else you cannot call it with `wa 6T magnet`)